src/write_prometheus.c

```diff
@@ -446,7 +446,8 @@ static void format_metric_families(strbuf_t *buf,
   }
 }
 
-static void format_text(strbuf_t *buf) {
+/* visible for testing */
+void format_text(strbuf_t *buf) {
```

Comment:
IMHO it would be much nicer for `write_prometheus_test.c`  to include `write_prometheus.c` (with `Makefile.am` updated accordingly), instead of changing all these functions to non-static.

Reply:
✅ done